doc/source/data/dataset-internals.rst

```diff
@@ -54,7 +54,7 @@ Datasets and Placement Groups
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 By default, Datasets configures its tasks and actors to use the cluster-default scheduling strategy ("DEFAULT"). You can inspect this configuration variable here:
-``ray.data.context.DatasetContext.get_current().scheduling_strategy``. This scheduling strategy will schedule these tasks and actors outside any present
+:meth:`ray.data.context.DatasetContext.get_current().scheduling_strategy <ray.data.context.DatasetContext.get_current>`. This scheduling strategy will schedule these tasks and actors outside any present
```

Comment:
Nit: Would linking to just the `DatasetContext` abstraction rather than the `get_current()` method be more appropriate? 🤔 
```suggestion
:class:`ray.data.context.DatasetContext.get_current().scheduling_strategy <ray.data.context.DatasetContext>`. This scheduling strategy will schedule these tasks and actors outside any present
```

Reply:
SG